app/models/publisher.rb

```diff
@@ -172,13 +181,19 @@ def generate_verification_token
     update_attribute(:verification_token, PublisherTokenRequester.new(publisher: self).perform)
   end
 
+  def clear_brave_publisher_id
```

Comment:
I don't think this method is used

Reply:
Ah good catch. This method was from an intermediate iteration and should have been abandoned. 👍